.travis.yml

```diff
@@ -39,10 +41,11 @@ install:
   - npm install -g jsonlint
 
 script:
-  - find . -type f -name "*.json" -print0 | xargs -0 -n1 jsonlint -q
-  - find . -type f -name "*.php" -print0 | xargs -0 -n1 php -l
-  - composer validate
+  - if [[ "$SNIFF" == "1" ]];then find . -type f -name "*.json" -print0 | xargs -0 -n1 jsonlint -q; fi
+  - if [[ "$SNIFF" == "1" ]];then find . -type f -name "*.php" -print0 | xargs -0 -n1 php -l; fi
+  - if [[ "$SNIFF" == "1" ]];then composer validate; fi
```

Comment:
As there is no committed `composer.lock`, you may want to validate against both the highest as well as the lowest PHP version.

Reply:
iirc composer validate only checks of the composer files are valid, independent of php version